doc/source/examples/rpt/tuning-parameters-with-nomad/tuning-parameters-with-nomad.rst

```diff
@@ -0,0 +1,320 @@
+
+
+:raw-html:`<br />`
+
+Running the simulation
+----------------------------------
+
+If it's the **first time** you're running this example, it is important to open the ``rpt_lethe_nomad.py`` script and complete the path to the ``rpt_3d`` executable on the :math:`14^{th}` line.
+
+.. attention::
+    If you don't complete the path to the ``rpt_3d`` executable on the :math:`14^{th}` line of the ``rpt_lethe_nomad.py`` script you won't be able to tune your parameters.
+
+Once you've completed the line mentioned above in the ``rpt_lethe_nomad.py`` script, you may run NOMAD by typing :
+
+.. code-block:: text
+
+    /home/myUserName/PathToNomad param_nomad.txt
+
+
+NOMAD will then execute the Python script (``rpt_lethe_nomad.py``) which is specified in the ``param_nomad.txt`` file. The Python script ``rpt_nomad_lethe.py`` proceeds the values of parameters to tune given by NOMAD, modifies the parameter file for Lethe, and runs the rpt_3d application. ``rpt_3d`` of Lethe executes the Monte Carlo ray model and calculates a cost function which is caught by NOMAD through the terminal. NOMAD executes its MADS algorithm and generates a new set of parameters until a terminating criterion is reached.
+
+
+:raw-html:`<br />`
+
+Results
+--------
+
+After running the optimization software, the best feasible solution will be displayed on the terminal.
+
+.. code-block:: text
+
+    A termination criterion is reached: No termination (all). Mesh minimum precision reached (Algo)
+
+    Best feasible solution:     #30212 ( 7.85479e-06 2.43045e+06 0.5002 )	Evaluation OK	 f =   0.03238789999999999725	 h =   0
+
+    Best infeasible solution:   Undefined.
+
+    Blackbox evaluations:        390
+    Total model evaluations:     39890
+    Cache hits:                  69
+    Total number of evaluations: 459
+
+.. tip::
+    Changing the initial values of the optimization problem to ones that are closer to the solution seen above can reduce the computation time.
+
+We may now verify if these values correspond to the studied system. To do so, as it was done in the previous example, we calculate the counts for the set of particle positions that we have.
+
+Before executing the ``rpt_3d`` application we have to change a few parameters in the ``rpt_parameters.prm`` file.
+
+First, in the *"rpt parameters"* subsection, we may set the ``verbosity`` parameter to ``verbose``, set the ``export counts`` parameter to ``true``, and change the name of the ``counts file`` that will be exported if we wish.
+
+Second, in the *"parameter tuning"* subsection, we have to set the ``tuning`` parameter to ``false`` since we're trying to tune parameters anymore.
+
+Lastly, in the *"detector parameters"* subsection, we have to change the values of the parameters that we tuned (``dead time``, ``activity``, and ``attenuation coefficient reactor``) with the ones NOMAD gave us.
+
+By doing the modifications mentioned above, the content ``rpt_parameters.prm`` should look like this :
```

Comment:
Instead of asking to modify the file, I suggest you give two prm files, one for the tuning and one for the simulation afterwards :)!

Reply:
Okay!
Should I still leave this, so that readers may see what are the differences between both files or is it not necessary? 
I would change the sentence structure telling them to use the second .prm file and then specifying the differences between the two .prm files.

Reply:
Perfect, that's what I was trying to say :)